### ISSUE TITLE
Add "local pages" for smart devices to build script

### ISSUE THREAD

**Issue by mariobehling** [fossasia/susi.ai#2495]
Please build pages for SUSI.AI smart devices on merged PRs in the branch "local-pages". Add it to the build script.

What are the differences? On the local deployment:
* On the frontpage, there should be a notification above the slider "You are currently accessing the local version of your SUSI.AI running on your smart device."
* The device section in the settings should have all the options to configure the device. (which is currently not entirely possible from the online web client)
* The skills CMS should use the locally stored skills on the local SUSI.AI server

**Comment by akshatnitd** [comment]
@mariobehling If we point the build to the local server, does that mean, the user has to create account again on the local server ? And other related info like Settings?

**Comment by mariobehling** [comment]
@akshatnitd Good point. We need to define and implement a policy how things need to work. E.g. local public skills should not be editable. Regarding sign up we need to think it through. For now the first step is to build the local pages and get the points in this issue working. Other issues need to be opened subsequently.

**Comment by akshatnitd** [comment]
Working on it. 
Will make the UI changes in a separate PR once #2500 is merged

**Comment by fragm3** [comment]
Assigned so contributors don't get confused
